lnd.go

```diff
@@ -443,6 +443,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 		return mkErr("error creating wallet config: %v", err)
 	}
 
+	rpcServer.server.cc.Wallet.Cfg.CoinSelectionStrategy =
```

Comment:
This is already set during the creation of the wallet.

Reply:
Ah I see now, Thanks!